src/bridges/zora/ZoraBridge.sol

```diff
@@ -183,7 +176,13 @@ contract ZoraBridge is BridgeBase {
                 sellerFundsRecipient,
                 0                     // Leave the finder fee as 0.
             );
-            return (0, 0, false);
+            
+            // Update the asset map to correlate new virtual token with existing
+            // bridge-owned nft.
+            nftAssets[_interactionNonce] = token;
```

Comment:
use `_updateVirtualAssetId` like you do with the cancel ask

Reply:
yup, updated!